03-transactions.md

```diff
@@ -158,22 +158,89 @@ Thus we use a simplified formula for *expected weight*, which assumes:
 
 The *expected weight* of a commitment transaction is calculated as follows:
 
-	transaction core: 4 + 1 + 1 + 4
-	transaction input: 32 + 4 + 1 + 4
-	transaction input witness: 1 + 74 + 74 + 1 + 1 + 34 + 34 + 1 + 1
-	transaction output: 8 + 1
-	transaction htlc output: 34
-	transaction to-local output: 34
-	transaction to-remote output: 25
-
+	p2wsh: 34 bytes
+		- OP_0: 1 byte
+		- OP_DATA: 1 byte (witness_script_SHA256 length)
+		- witness_script_SHA256: 32 bytes
+
+	p2wpkh: 22 bytes
+		- OP_0: 1 byte
+		- OP_DATA: 1 byte (public_key_HASH160 length)
+		- public_key_HASH160: 20 bytes
+
+	multi_sig: 71 bytes
+		- OP_2: 1 byte
+		- OP_DATA: 1 byte (pub_key_alice length)
+		- pub_key_alice: 33 bytes
+		- OP_DATA: 1 byte (pub_key_bob length)
+		- pub_key_bob: 33 bytes
+		- OP_2: 1 byte
+		- OP_CHECKMULTISIG: 1 byte
+
+	witness: 222 bytes
```

Comment:
Seems like this is actually calculating the `weight` instead of the raw size in bytes.